pyrq/pools.py

```diff
@@ -0,0 +1,226 @@
+import time
+import socket
+import os
+
+from pyrq import helpers
+
+DEFAULT_CHUNK_SIZE = 10
```

Comment:
navysil jsem to na 100, nicmene rozpad na jednotlive casti ma v PHP jiny vyznam.. Tady bych to tak nedelal, je to dost zbytecne.. btw: pouziva to v PHP vubec nekdo?